Purpose: Avoid compiler warning about unused variable when SE Linux is not
         enabled.
Authour: Petter Reinholdtsen
Fixes:   -
Status:  Not yet submitted upstream.

--- a/src/init.c
+++ b/src/init.c
@@ -2615,7 +2615,9 @@
 	char			*p;
 	int			f;
 	int			isinit;
+#ifdef WITH_SELINUX
 	int			enforce = 0;
+#endif
 
 	/* Get my own name */
 	if ((p = strrchr(argv[0], '/')) != NULL)
